File: model/src/main/java/ar/edu/itba/paw/model/ActorRole.java
package ar.edu.itba.paw.model;

import java.util.Collections;
import java.util.LinkedList;
import java.util.List;

public class ActorRole {

    private List<SeriesCharacter> roles = Collections.emptyList();
    private Actor actor;
    private Series series;

    public ActorRole(Actor actor, Series series) {
        this.actor = actor;
        this.series = series;
    }

    public ActorRole() {}

    public List<SeriesCharacter> getRoles() {
        return roles;
    }

    public void addRole(String role) {
        SeriesCharacter sc = new SeriesCharacter();
        sc.setName(role);
        roles.add(sc);
    }

    public Actor getActor() {
        return actor;
    }

    public void setActor(Actor actor) {
        this.actor = actor;
    }

    public Series getSeries() {
        return series;
    }

    public void setSeries(Series series) {
        this.series = series;
    }
}

File: apiCrawlerUtility/ApiCrawlerTuTV/Model/ActorRole.cs
using ApiCrawlerTuTV.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ApiCrawlerTuTV {
    public class ActorRole {
        public int id { get; set; }
        public int tvDbId { get; set; }
        public String role { get; set; }
        public Actor actor { get; set; }
        public Series series { get; set; }
        public string imageUrl { get; set; }
    }

}
